src/slang-printers/print-comments.ts

```diff
@@ -13,9 +17,10 @@ function isPrintable(comment: Comment): boolean {
 }
 
 export function printComments(
-  path: AstPath<AstNode>,
+  path: AstPath<StrictAstNode>,
   options: ParserOptions<AstNode>
 ): Doc[] {
+  if (typeof path.node.comments === 'undefined') return [];
```

Comment:
This is a weird pattern to see nowadays. `typeof x === 'undefined'` made sense in old javascript where `x` could be undeclared, but I think today you just use `x === undefined`, even more so in typescript.